### ISSUE TITLE
China, DF-26, CEP way too large? except ASBM version

### ISSUE THREAD

**Issue by Luigi-II** [PygmalionOfCyprus/cmo-db-requests#5183]
### DB Selector

DB3K

### Affected DBID(s)

3373 3374 3375 3376 4315 4316

### Summary of Changes

- [ ] Suggest setting the CEP to **20m** rather than 300m
- [ ] Add **Beidou/COMPASS** to their properties

I do see a number of sources claiming that "analysis puts the DF-26's CEP between 150m and 450m". Perhaps as a third-party estimate, I can't say anything about it, but such CEP performance makes the DF-26 basically usable as some kind of "nuclear warhead IRBM" (some analyses have also considered it a nuclear warhead carrier, which may be the source of the error).

In fact, this CEP arrangement and estimate is completely at odds with some of the other analyses available and its reasonable use inferences. I note that in the sources describing the DF-21, its accuracy is even thought to be up to 10m CEP with the Beidou navigation system and active radar. The DF-26 is described only as "navigation system unknown". Wouldn't the Chinese use the Beidou navigation system they already use on the DF-21 variant on the more advanced DF-26? Obviously not.

In the absence of RF seeker, I think 20m CEP is reasonable based on the accuracy of Beidou navigation system alone. This is same as DF-27 conventional in game. And do notice, its payload is an MaRV, which certainly has the ability to make a correction at the terminal phase to achieve a more accurate strike.

### Sources

https://en.wikipedia.org/wiki/DF-21#See_also

> Duncan Lennox, ed., Jane's Strategic Weapon Systems, 51st ed. (Coulsdon, Surrey, U.K.:Jane's Information Group, 2010)

![image](https://github.com/user-attachments/assets/7f02565e-4fae-4fad-8c02-6cfbbd20c0a0)
The CEP of the existing DF-26 variant is at the ICBM level
![image](https://github.com/user-attachments/assets/568e2557-4084-4dfc-bdb4-2c3260d0da74)

**Comment by yU2187sie** [comment]
According to the information I can check, the DF-26 is a new missile developed at the same time as the DF-21D. It's not an improvement, it's a completely new platform, and it fits the Chinese description of "starting from scratch" (compared to the DF-21D).
It was developed with a large amount of advanced technology available to China at the time, including guidance equipment. At the same time, its anti-ship capability was a requirement from the beginning of the design. Compared with the DF-21, this vehicle of the last century has a natural advantage in anti-ship capability, so its accuracy index has higher requirements than the DF-21. We also know that compared to the DF-21D, the DF-26 is a more reliable ASBM for the PLA (the number of DF-21D PLA equipment is really a little small...).

**Comment by Luigi-II** [comment]
> According to the information I can check, the DF-26 is a new missile developed at the same time as the DF-21D. It's not an improvement, it's a completely new platform, and it fits the Chinese description of "starting from scratch" (compared to the DF-21D). It was developed with a large amount of advanced technology available to China at the time, including guidance equipment. At the same time, its anti-ship capability was a requirement from the beginning of the design. Compared with the DF-21, this vehicle of the last century has a natural advantage in anti-ship capability, so its accuracy index has higher requirements than the DF-21. We also know that compared to the DF-21D, the DF-26 is a more reliable ASBM for the PLA (the number of DF-21D PLA equipment is really a little small...).

Can you provide related info?

**Comment by PygmalionOfCyprus** [comment]
I definitely agree that the CEP should be bumped down. I went with the "analyst" number but it looks like even CSIS has upgraded its estimate to 100m (although it still quotes the 150-450m range in its article).

There may be differences between the missiles not immediately related to guidance and not obvious to us that result in the CEP difference. It's also possible that you're correct and the only reason we haven't seen smaller numbers is because analyzed tests have been unusually unsuccessful, etc. On the other hand, the DF-21's seeker may be doing a lot more heavy lifting than we know, resulting in its lower CEP.

I've implemented the GNSS guidance as suggested and knocked the CEP on the non-AShM variants to 100m. Will keep an eye on this thread for further debate and in case we get more concrete info.
